src/zcbenchmarks.cpp

```diff
@@ -291,11 +291,11 @@ double benchmark_try_decrypt_notes(size_t nAddrs)
     }
 
     auto sk = libzcash::SpendingKey::random();
-    auto tx = GetValidReceive(*pzcashParams, sk, 10, true);
+    auto walletTx = GetValidReceive(*pzcashParams, sk, 10, true);
 
     struct timeval tv_start;
     timer_start(tv_start);
-    auto nd = wallet.FindMyNotes(tx);
+    auto nd = wallet.FindMyNotes(walletTx.getWrappedTx());
```

Comment:
nd does not seem to be used? (It was the same in the original code anyway)

Reply:
It's a benchmark, so I believe they want to verify the time it takes for finding notes